app/src/main/java/com/google/samples/apps/sunflower/data/GardenPlantingRepository.kt

```diff
@@ -16,7 +16,9 @@
 
 package com.google.samples.apps.sunflower.data
 
-class GardenPlantingRepository private constructor(
+import javax.inject.Inject
+
+class GardenPlantingRepository @Inject constructor(
```

Comment:
The code you removed below was to obtain this class as a singleton. Do the same by annotating this with `@Singleton`